app/views/cases/index.html.erb

```diff
@@ -6,22 +6,20 @@
       <th>Location</th>
       <th>Status</th>
       <th colspan="1"></th>
+    </tr>
   </thead>
-   </tr>
   <tbody>
     <% @cases.each do |case1| %>
       <tr>
-          <td><%= case1.case_title %></td>
-          <td><%= case1.location%></td>
-          <td><%=case1.status%></td>
-          <td><%= link_to case_path(case1) do%><%= image_tag "detail.png", size: "20x40" ,class: "some class here",alt: "show" %>
-<% end %><td>
-          <td><%= link_to edit_case_path(case1) do%><%= image_tag "edit.png", size: "20x40" ,class: "some class here",alt: "Edit" %><%end%></td>
-
-          <td><%= link_to case1, method: :delete, data: { confirm: 'Are you sure?' } do %><%= image_tag "del.png", size: "20x40" ,class: "some class here",alt: "Edit" %><%end%></td>
-        
-      </tr>
-     <% end %>
+        <td><%= case1.case_title %></td>
+        <td><%= case1.location%></td>
+        <td><%=case1.status%></td>
+        <td><%= link_to case_path(case1) do%><%= image_tag "detail.png", size: "20x40" ,class: "some class here",alt: "show" %>
+        <% end %><td>
+        <td><%= link_to edit_case_path(case1) do%><%= image_tag "edit.png", size: "20x40" ,class: "some class here",alt: "Edit" %><%end%></td>
+        <td><%= link_to case1, method: :delete, data: { confirm: 'Are you sure?' } do %><%= image_tag "del.png", size: "20x40" ,class: "some class here",alt: "Edit" %><%end%></td>
```

Comment:
Replace bootstrap theme icons instead of image_tag. Please remove image_tag and use only theme icon CSS classes